clang/test/CodeGenSYCL/wg_init.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
+#include "Inputs/sycl.hpp"
+
+template <typename KernelName, typename KernelType>
```

Comment:
Please use the mock headers and add a comment describing what test is doing as per guidelines for FE tests here - https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md

Reply:
Thanks change made.

Reply:
Sorry, but I think there is some misunderstanding. What @smanna12 meant is that you need to use `handler::parallel_for_work_group` function defined in `test/CodeGenSYCL/Inputs/sycl.hpp` (which is already included by your test) instead of defining `kernel_parallel_for_work_group` in your test. We also usually don't add LLVM license header in test files.

Reply:
OK thanks for clarification. Will do!

Reply:
Change made

Reply:
Thank you.